packages/scss/src/components/tag/component.scss

```diff
@@ -1,13 +1,13 @@
-@mixin component {
+@mixin component($atRoot: 'without: rule') {
 	background-color: var(--components-tag-background);
 	border-radius: var(--commons-borderRadius-M);
-	font-size: var(--components-tag-fontSize);
```

Comment:
Couldn't this be dangerous in case a tag is called in a parent component were a font-size / line-height is applied?

Reply:
It's moved line 22.